tooling/car/unixfs.go

```diff
@@ -1,21 +1,35 @@
 package car
 
 import (
+	"bytes"
 	"context"
 	"fmt"
 	"os"
 	"path"
 	"strings"
 
+	"github.com/ipfs/boxo/blockservice"
+	"github.com/ipfs/boxo/ipld/car/v2/blockstore"
+	"github.com/ipfs/boxo/ipld/merkledag"
+	"github.com/ipfs/boxo/ipld/unixfs/io"
 	"github.com/ipfs/gateway-conformance/tooling/fixtures"
-	"github.com/ipfs/go-blockservice"
 	"github.com/ipfs/go-cid"
 	format "github.com/ipfs/go-ipld-format"
-	"github.com/ipfs/go-merkledag"
-	"github.com/ipfs/go-unixfs/io"
-	"github.com/ipld/go-car/v2/blockstore"
+	"github.com/ipld/go-ipld-prime"
+	_ "github.com/ipld/go-ipld-prime/codec/cbor"
+	_ "github.com/ipld/go-ipld-prime/codec/dagcbor"
+	_ "github.com/ipld/go-ipld-prime/codec/dagjson"
+	_ "github.com/ipld/go-ipld-prime/codec/json"
+	"github.com/ipld/go-ipld-prime/multicodec"
+	mc "github.com/multiformats/go-multicodec"
 )
 
+func init() {
+	// Note we force imports of dagcbor and dagjson above.
+	// They are registering themselves with the multicodec package
+	// during their `init()`.
+}
```

Comment:
Do we still need the init here?

Reply:
Fixed, moved to a comment above the imports